tvaultuiv2/src/views/public/HomePage/index.js

```diff
@@ -399,6 +399,7 @@ const LoginPage = () => {
     axios
       .post(`${config.url}/auth/oidc/auth_url`, payload)
       .then((res) => {
+        console.log('res', res);
```

Comment:
Please remove console
```suggestion
        console.log('res', res);
```